pypeit/scripts/coadd_2dspec.py

```diff
@@ -221,6 +221,8 @@ def main(args):
                                                           vel_type=None,
                                                           maskdef_designtab=maskdef_designtab)
 
+        all_spec2d['meta']['exptime'] = coadd.exptime_coadd
```

Comment:
Can we rename this parameter here and in the documentation as an effective exposure time or something along those lines. It is not a real exposure time right? I think that would make it clearer.

Reply:
Done